deps.edn

```diff
@@ -67,19 +67,19 @@
                         :extra-deps
                         {clj.native-image/clj.native-image
                          {:git/url "https://github.com/taylorwood/clj.native-image.git"
-                          :sha     "7708e7fd4572459c81f6a6b8e44c96f41cdd92d4"}}}
+                          :sha     "4604ae76855e09cdabc0a2ecc5a7de2cc5b775d6"}}}
 
            ;; Checks
 
            :benchmark {:extra-paths ["benchmark/src"]
                        :extra-deps {clj-http/clj-http {:mvn/version "3.12.3"}
-                                    org.clojure/tools.cli {:mvn/version "1.0.206"}
+                                    org.clojure/tools.cli {:mvn/version "1.0.214"}
                                     incanter/incanter-core {:mvn/version "1.9.3"}
                                     incanter/incanter-charts {:mvn/version "1.9.3"}}
                        :main-opts ["-m" "benchmark.cli"]}
 
-           :format {:extra-deps {cljfmt/cljfmt {:mvn/version "0.8.0"}}
+           :format {:extra-deps {cljfmt/cljfmt {:mvn/version "0.9.2"}}
                     :main-opts ["-m" "cljfmt.main" "check"]}
 
-           :outdated {:extra-deps {com.github.liquidz/antq {:mvn/version "2.2.983"}}
-                      :main-opts ["-m" "antq.core"]}}}
+           :outdated {:extra-deps {com.github.liquidz/antq {:mvn/version "2.2.992"}}
```

Comment:
It would be nice to run this either in the ci pipeline or as a regular task somehow.

Reply:
@TimoKramer, any insights on this?

Reply:
I don't see how we would be running this during CI. It would fail all the time and updating deps shouldn't be part of each random PR but should be it's own PR, I think. It would be much better to have kind of a bot to open PRs on this. Maybe I can build something...

Reply:
https://github.com/nnichols/clojure-dependency-update-action/blob/master/dependency-check.sh

Reply:
By now, I would also tend to rather have a separate task for this than to require an update with each PR. 

It was annoying enough to try to do it perfectly in this PR and I can imagine it would be even more annoying if there were major code changes involved that you wanted to get in asap.

Reply:
Can we use GitHub Actions alongside Circleci?

Reply:
sure

Reply:
Nice, then it shouldn't be too hard to set this up.